slack-archive/src/main/java/juja/microservices/slack/archive/service/impl/ArchiveServiceImpl.java

```diff
@@ -28,9 +31,16 @@ public ArchiveServiceImpl(ArchiveRepository repository) {
     }
 
     @Override
-    public List<String> getChannels() {
-        //TODO Should be implemented
-        return null;
+    public List<ChannelDTO> getChannels() {
+        List<ChannelDTO> channels = repository.getChannels().stream()
+                .map(this::convertChannelToChannelDTO)
```

Comment:
.map(c -> new ChannelDTO(c.getChannelId(), c.getChannelName()))
a bit shorter

Reply:
Yes, of course, but if we add additional fields into ChannelDTO in the future it will be a little bit longer. So I made separate method for converting in advance for this case.